rom/dev/tests/rom_integration_tests/test_fmcalias_derivation.rs

```diff
@@ -472,12 +476,11 @@ fn test_pcr_log_across_update_reset() {
         .owner_pubkey_digest(&image_bundle.manifest.preamble)
         .unwrap();
 
-    const FMC_SVN: u32 = 2;
-    const FMC_FUSE_SVN: u32 = 1;
+    const FW_SVN: u32 = 2;
 
     let fuses = Fuses {
         anti_rollback_disable: false,
-        fmc_key_manifest_svn: FMC_FUSE_SVN,
+        fw_svn: [1, 0, 0, 0],
```

Comment:
same comment about just defining the constant in one place.

Reply:
Ditto.